### PR TITLE
added tasks

### PR DESCRIPTION
**Pull request by dmitrikll**

### CODE REVIEW

File: src/test/java/org/example/Task01Test.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package org.example;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class Task01Test {
+
+    @Test
+    public void testCountJewelsInStones() {
+        Task01 testCount = new Task01();
```
Comment (mkharkhuta) [inline src/test/java/org/example/Task01Test.java:10]: Bad naming, it's not cleare what is testing 
It would be better to give reasonable names: 
Task01 -> JewelsOnStonesSolution
testCountJewelsInStones() -> countJewelsInStones_StonesAreInJewels_CoutedOk, or something using given when then rule

File: src/test/java/org/example/Task01Test.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+        Assert.assertEquals(3, testCount.countJewelsInStones("aA", "aAAbbbb"));
```
Comment (mkharkhuta) [inline src/test/java/org/example/Task01Test.java:12]: Consider using assetThat(actual).isEqualTo(expected)

File: src/test/java/org/example/Task01Test.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        Assert.assertEquals(0, testCount.countJewelsInStones("", ""));
+        Assert.assertEquals(0, testCount.countJewelsInStones("aa", "AA"));
+        Assert.assertEquals(6, testCount.countJewelsInStones("acd", "aaabbbccc"));
+    }
```
Comment (mkharkhuta) [inline src/test/java/org/example/Task01Test.java:16]: Split it to different tests - positive, negative, edge cases

**Review by mkharkhuta — COMMENTED** [review]
Good job, but requires test refactoring
